src/main/resources/static/views/investment/Investments.html

```diff
@@ -0,0 +1,46 @@
+<div class="container">
+
+    <div class="well" style="max-height: 210px; overflow: auto;">
+        <ul ng-repeat="investment in investments"
+            class="list-group checked-list-box">
+            <li class="list-group-item {{investment.paid ? 'list-group-item-success' :
+        (investment.verified ? 'list-group-item-warning' : 'list-group-item-danger')}}"
+                data-color="">
+                <div class="row">
+                    <div class="col-xs-3">
+                        {{investment.id}}
+                        <a href="#">
+                            {{investment.investor.name}}
+                        </a>
+                    </div>
+                    <div class="col-xs-4">
+                        <a href="#">
+                            {{ investment.project.desc.name}}
+                        </a>
+                    </div>
+                    <div class="col-md-3 hidden-sm hidden-xs">
+
+                        {{investment.dateOfInvestment.year}}
+                        {{investment.dateOfInvestment.month}}
+                        {{investment.dateOfInvestment.dayOfMonth}},
+                        {{investment.dateOfInvestment.hour}}:{{investment.dateOfInvestment.minute}}
+                    </div>
+                    <div class="col-xs-2">
+                        {{investment.sum | currency}}
+                    </div>
+                </div>
+            </li>
+        </ul>
+    </div>
+</div>
+<!--
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)